<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>LoginDialog</class>
	<widget class="QDialog" name="LoginDialog">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>400</width>
				<height>550</height>
			</rect>
		</property>
		<layout class="QVBoxLayout" name="mainLayout">
			<property name="spacing">
				<number>0</number>
			</property>
			<property name="leftMargin">
				<number>40</number>
			</property>
			<property name="topMargin">
				<number>40</number>
			</property>
			<property name="rightMargin">
				<number>40</number>
			</property>
			<property name="bottomMargin">
				<number>40</number>
			</property>
			<item>
				<widget class="QLabel" name="label_title">
					<property name="text">
						<string>BOARD GAME</string>
					</property>
					<property name="alignment">
						<set>Qt::AlignmentFlag::AlignCenter</set>
					</property>
				</widget>
			</item>
			<item>
				<widget class="QStackedWidget" name="stackedWidget">
					<property name="currentIndex">
						<number>0</number>
					</property>
					<widget class="QWidget" name="loginPage">
						<layout class="QVBoxLayout" name="loginLayout">
							<property name="spacing">
								<number>10</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>20</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QLabel" name="label_user_login">
									<property name="text">
										<string>Username</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLineEdit" name="editUsername">
									<property name="placeholderText">
										<string>Enter account</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_pwd_login">
									<property name="text">
										<string>Password</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLineEdit" name="editPassword">
									<property name="echoMode">
										<enum>QLineEdit::EchoMode::Password</enum>
									</property>
									<property name="placeholderText">
										<string>Enter password</string>
									</property>
								</widget>
							</item>
							<item>
								<spacer name="spacer1">
									<property name="orientation">
										<enum>Qt::Orientation::Vertical</enum>
									</property>
									<property name="sizeHint" stdset="0">
										<size>
											<width>20</width>
											<height>20</height>
										</size>
									</property>
								</spacer>
							</item>
							<item>
								<widget class="QPushButton" name="btnLogin">
									<property name="text">
										<string>LOGIN</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="btnSwitchToRegister">
									<property name="text">
										<string>CREATE ACCOUNT</string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
					<widget class="QWidget" name="registerPage">
						<layout class="QVBoxLayout" name="registerLayout">
							<property name="spacing">
								<number>10</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>20</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QLabel" name="label_user_reg">
									<property name="text">
										<string>New Username</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLineEdit" name="editRegUsername">
									<property name="placeholderText">
										<string>Pick a unique name</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_pwd_reg">
									<property name="text">
										<string>Password</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLineEdit" name="editRegPassword">
									<property name="echoMode">
										<enum>QLineEdit::EchoMode::Password</enum>
									</property>
									<property name="placeholderText">
										<string>At least 6 characters</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLabel" name="label_pwd_confirm">
									<property name="text">
										<string>Confirm Password</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLineEdit" name="editRegConfirm">
									<property name="echoMode">
										<enum>QLineEdit::EchoMode::Password</enum>
									</property>
									<property name="placeholderText">
										<string>Repeat password</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="btnExecuteRegister">
									<property name="text">
										<string>REGISTER NOW</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="btnBackToLogin">
									<property name="text">
										<string>BACK TO LOGIN</string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</widget>
			</item>
			<item>
				<widget class="QLabel" name="lblStatus">
					<property name="text">
						<string/>
					</property>
					<property name="alignment">
						<set>Qt::AlignmentFlag::AlignCenter</set>
					</property>
				</widget>
			</item>
		</layout>
	</widget>
	<resources/>
	<connections/>
</ui>
